helm/examples/azure/values-auto-azurefile.yaml

```diff
@@ -3,24 +3,48 @@
 # provisioned Persistent Volume's with a reclaimPolicy of "Delete"
 
 storage:
-  dllStorage:
+  planes:
+  - name: dali
+    storageSize: 1Gi
+    storageClass: "azurefile"
+    prefix: "/var/lib/HPCCSystems/dalistorage"
+    labels:
+    - dali
+
+  - name: dll
+    prefix: "/var/lib/HPCCSystems/queries"
     storageSize: 3Gi
```

Comment:
incidental, I don't know why we have 3Gi for dll and 1Gi for data in our examples.
I changed all to 1Gi it in our stock values.yaml